### ISSUE TITLE
simplify installation instructions

### ISSUE THREAD

**Issue by haraldschilly** [abelfunctions/abelfunctions#140]
you could add this one-liner to simplify the installation:

    sage -pip install git+https://github.com/abelfunctions/abelfunctions.git

and maybe also add it to the [pypi index](https://pypi.python.org/pypi?%3Aaction=search&term=abelfunctions&submit=search)?

**Comment by rparini** [comment]
I agree the package should be on pypi. I can write the github action to build and push to pypi when a git tag is pushed (or github Release made) of the form `vX.Y.Z`. We'd need an API token for a pypi account stored as a [GitHub secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#about-encrypted-secrets).

@cswiercz, if that sounds good to you, I could use the pypi API token for my account and add you as a co-owner on pypi once the package is created there, or if you could create a github secret with your own API token so that you're the sole owner (you could always add me as a Maintainer if you like)

We can upload to https://test.pypi.org/ first to check everything looks good before switching to the real pypi

**Comment by cswiercz** [comment]
@rparini Thanks for reviving this ticket. I think it's a good idea, especially to make it easier for people to use the package.

If you don't mind, I'll add you as a co-owner to my own token/account. Pushing something to pypi is something that I should've learned how to do a long time ago!

**Comment by rparini** [comment]
@cswiercz sure, no problem
